go.mod

```diff
@@ -50,3 +50,9 @@ replace (
 )
 
 go 1.13
+
```

Comment:
Why we need this here?

Reply:
Yeah... let's figure out why this is appearing, as we are trying to keep our dependencies lean... in particular the sriov-controller is weird, as it's not used by anything anywhere.

Reply:
Those new 'require' lines are result of running unit tests (scripts/run_unit_tests.sh)
- [ ] remove unnecessary changes in go.sum/go.mod